internal_displacement/interpreter.py

```diff
@@ -413,18 +434,23 @@ def extract_locations(self, sentence, root=None):
         else:
             return []
 
-    def extract_dates(self, sentence, story, root=None):
+    def extract_dates(self, sentence, story, publication_date=None, root=None):
```

Comment:
I have a feeling we can simplify this function a bit more. I did some testing and found I actually seemed to get better results by just using DATE entities extracted from the sentence using Spacy. What do you think?